rootfs-builder/rootfs.sh

```diff
@@ -580,7 +580,7 @@ EOT
 			[ -n "${AGENT_VERSION}" ] && git checkout "${AGENT_VERSION}" && OK "git checkout successful"
 			make clean
 			make
-			make install DESTDIR="${ROOTFS_DIR}"
+			make install DESTDIR="${ROOTFS_DIR}" INIT="${AGENT_INIT}"
```

Comment:
Thought: The new agent doesn't (yet) support `SECCOMP` but it might be worth adding a comment stating that (which would explain the inconsistency between the go and rust agent `make install` commands.

Reply:
okay~~ update asap. ;)